bliss/catalog.py

```diff
@@ -422,6 +488,63 @@ def __init__(self, height: int, width: int, d: Dict[str, Tensor]) -> None:
         assert self.n_sources.shape == (self.batch_size,)
         super().__init__(**d)
 
+    def _test_tensor_all_close(self, left, right):
+        return torch.allclose(left, right, atol=1e-4)
```

Comment:
It's better to define this method outside of both catalog classes so it doesn't have to be duplicated. If we even need it. Could we just call `torch.allclose` directly with the default `atol` instead of calling `_test_tensor_all_close`? The code would be more readable if we could avoid one level of indirection.

Reply:
I move it to test_dc2.py in commit a521a09.